04/app.js

```diff
@@ -1,9 +1,29 @@
 const user = {
-    firstName: 'Adam',
-    lastName: 'Nowak',
-    born: {
-        day: '14',
-        month: '04',
-        year: '1985'
-    }
-}
\ No newline at end of file
+	firstName: 'Adam',
+	lastName: 'Nowak',
+	born: {
+		day: '04',
+		month: '04',
+		year: '1985',
+	},
+};
+
+const fullDate = getCurrentDate();
+
+function getCurrentDate() {
+	const now = new Date();
+	return now;
+}
+
+function isBirthdatToday(now, obj) {
+	const currentlyMonth = now.getMonth() + 1;
+	const currentlyDay = now.getDate();
+
+	if (currentlyDay == obj.born['day'] && currentlyMonth == obj.born['month']) {
```

Comment:
Warto używać ścisłego porównania tj. `===` - nie jest wykonywana niejawna konwersja i mamy większa kontrolę nad kodem. 
Niestety wtedy też trzeba uważać na to, że `'04'` nie jest równe `'4'`